tests/integration/hooks/__testFiles__/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 iteratec GmbH
```

Comment:
File name should be specific to finding post process, as this is in a generic folder